language/move-prover/tests/sources/functional/aborts-if.exp

```diff
@@ -8,8 +8,6 @@ error:  A postcondition might not hold on this return path.
     │
     =     at tests/sources/functional/aborts-if.move:32:5: abort2_incorrect (entry)
     =     at tests/sources/functional/aborts-if.move:32:5: abort2_incorrect (exit)
-    =         _x = <redacted>,
```

Comment:
Looks like all debug info is gone here. It's likely OK we submit this, but we need to get this back ASAP, given that its needed for folks working on the specs.

Reply:
Agreed.  Debug info and the other test you commented on are the two major outstanding issues once this PR lands.